src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -15,10 +16,27 @@ import spray.routing._
 import scala.reflect.runtime.universe._
 import scala.util.{Failure, Success, Try}
 
-
-object CromwellApiServiceActor {
-  def props(workflowManagerActorRef : ActorRef, swaggerService: SwaggerService): Props = {
-    Props(new CromwellApiServiceActor(workflowManagerActorRef, swaggerService))
+object SwaggerService {
+  /*
+    Because of the implicit arg requirement apply() doesn't work here, so falling back to the less
+    idiomatic (but not unheard of) from().
+   */
```

Comment:
For the sake of others, it shows up a lot in google based APIs and those who enjoy them. I'm not a huge fan in scala because of apply but IMO it's a reasonable fallback when apply isn't appropriate.